client/src/commons/Forms/PlacesTypesAndSubTypes/usePlacesTypesAndSubTypes.ts

```diff
@@ -17,6 +18,11 @@ const usePlacesTypesAndSubTypes = (parameters: usePlacesTypesAndSubTypesIncome)
         axios.get('/intersections/getPlacesSubTypesByTypes').then(
             result => {
                 if (result && result.data) {
+
+                    for(let placeTypes in result.data){
```

Comment:
change to forEach

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
JSON אי אפשר כי זה לא מערך, זה

Reply:
Can't because it's not an array, it's JSON

Reply:
Object.keys.forEach?

Reply:
👍